onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/beans/StopScheduleBeanServiceImpl.java

```diff
@@ -233,9 +233,14 @@ public StopCalendarDaysBean getCalendarForStop(AgencyAndId stopId) {
         stiBean.setDepartureTime(sti.getDepartureTime());
         stiBean.setServiceId(AgencyAndIdLibrary.convertToString(serviceId));
 
-        stiBean.setArrivalEnabled(bst.getBlockSequence() > 0);
-        stiBean.setDepartureEnabled(bst.getBlockSequence() + 1 < blockConfig.getStopTimes().size());
-
+        boolean firstStopTimeInBlock = ( bst.getBlockSequence() == 0 );
```

Comment:
Nit - space after opening parenthesis `( ` and before closing parenthesis ` )` should be remove for each of the added lines here.